servers/zms/src/main/java/com/yahoo/athenz/zms/DBService.java

```diff
@@ -200,7 +200,7 @@ void auditLogRequest(ResourceContext ctx, String domainName, String auditRef,
     
     Domain makeDomain(ResourceContext ctx, String domainName, String description, String org,
             Boolean auditEnabled, List<String> adminUsers, String account, int productId,
-            List<String> solutionTemplates, String auditRef) {
+            List<String> solutionTemplates, String auditRef, String applicationId) {
```

Comment:
Can we please move applicationId argument to be after productId. It'd like to be consistent and have auditRef/caller as last arguments in our APIs.

Reply:
done